src/check_numerics.cpp

```diff
@@ -33,7 +33,7 @@
 
 namespace miopen {
 
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_CHECK_NUMERICS)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_CHECK_NUMERICS, uint64_t, 0)
```

Comment:
this seems like a boolean flag given its use

Reply:
MIOPEN_CHECK_NUMERICS functions like a bitmask.

Reply:
When I grep for uses, it seems to act more like a `boolean` flag. I'm not insisting on this change or the `enum` change. We can address these with `TODOs` now and follow up PRs (for me).

Reply:
Thank you. I missed that because our usage apparently looked like boolean logic.